examples/sdk-app-example/app/src/main/java/com/example/bugsnag/android/ExampleApplication.kt

```diff
@@ -26,7 +26,7 @@ class ExampleApplication : Application() {
         super.onCreate()
 
         val config = Configuration.load(this)
-        config.setUser("123456", "joebloggs@example.com", "Joe Bloggs")
+        config.setUser("123456", "joebloggs@example.com", "Joe Bzloggs")
```

Comment:
Just checking that this is intentional and not a stray key touch?

Reply:
that was definitely a stray key